systemd/CMakeLists.txt

```diff
@@ -1,3 +1,12 @@
+include(GNUInstallDirs)
+
+configure_file(${CMAKE_SOURCE_DIR}/systemd/ceph-osd@.service.in
+  ${CMAKE_BINARY_DIR}/systemd/ceph-osd@.service @ONLY)
```

Comment:
I was trying to separate `configure_file()` and `install()` to explicitly doing to config file rendering and installation. Do you think it improves some readability?